cmd/res/profiles/llrp.impinj.profile.yaml

```diff
@@ -0,0 +1,153 @@
+name: "LLRP-Impinj-Profile"
+manufacturer: "Impinj"
+labels:
+  - "LLRP"
+  - "RFID"
+  - "Impinj"
+description: "A device profile with added Impinj support."
+
+deviceResources:
+  - 
+    name: "ImpinjCustomExtensionMessage"
+    properties:
+      valueType: "String"
+      readWrite: "RW"
+      defaultValue: "AAAAAA=="
+    attributes:
+      vendor: "25882"
+      subtype: "21"
+
+  - 
+    name: "ImpinjSaveSettingsMessage"
+    properties:
+      valueType: "String"
+      readWrite: "W"
+    attributes:
+      vendor: "25882"
+      subtype: "23"
+
+  - 
+    name: "ReaderCapabilities"
+    description: "The capabilities of a reader"
+    properties:
+      valueType: "Object"
+      readWrite: "R"
+
+  - 
+    name: "ReaderConfig"
+    description: "A Reader's configuration"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "ROSpecID"
+    description: "Client-generated Reader Operation Specification Identifier"
+    properties:
+      valueType: "uint32"
+      readWrite: "W"
+      defaultValue: "1"
+
+  - 
+    name: "AccessSpecID"
+    description: "Client-generated Access Specification Identifier"
+    properties:
+      valueType: "uint32"
+      readWrite: "W"
+      defaultValue: "1"
+      value: { type: "uint32", readWrite: "W", defaultValue: "1" }
+
+  - 
+    name: "ROSpec"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "AccessSpec"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "Action"
+    properties:
+      valueType: "String"
+      readWrite: "W"
+
+  - 
+    name: "ROAccessReport"
+    properties:
+      valueType: "Object"
+      readWrite: "R"
+
+  - 
+    name: "ReaderEventNotification"
+    properties:
+      valueType: "Object"
+      readWrite: "R"
```

Comment:
Sure. I assume DS gives an error if try to do a GET command for these?

Reply:
I just looked an yes, it will send this:
```go
return nil, errors.Errorf("unknown resource type: %q", reqs[i].DeviceResourceName)
```
I was mostly commenting about it matching the other profile as well. the non-impinj profile had the async comments, but this one did not.